File: php-sdk/src/triplink/vcc/card/client/query/QueryCloseResultRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: wang.xiaoliang
 * Date: 2021/8/2
 * Time: 18:56
 */

namespace triplink\vcc\card\client\query;

class QueryCloseResultRequest
{
    /**
     * the request number of the original update request number
     */
    var $requestId;
    /**
     * the request time of the original update request time
     */
    var $requestTime;
    /**
     * the merchant name of the original update merchant name
     */
    var $merchantName;
    /**
     * request extra info
     * json type
     * eg: {"BinFlagDcre":"12345"}
     */
    var $reqExtra;
    /**
     * operator
     */
    var $operator;

    /**
     * @return mixed
     */
    public function getRequestId()
    {
        return $this->requestId;
    }

    /**
     * @param mixed $requestId
     */
    public function setRequestId($requestId): void
    {
        $this->requestId = $requestId;
    }

    /**
     * @return mixed
     */
    public function getRequestTime()
    {
        return $this->requestTime;
    }

    /**
     * @param mixed $requestTime
     */
    public function setRequestTime($requestTime): void
    {
        $this->requestTime = $requestTime;
    }

    /**
     * @return mixed
     */
    public function getMerchantName()
    {
        return $this->merchantName;
    }

    /**
     * @param mixed $merchantName
     */
    public function setMerchantName($merchantName): void
    {
        $this->merchantName = $merchantName;
    }

    /**
     * @return mixed
     */
    public function getReqExtra()
    {
        return $this->reqExtra;
    }

    /**
     * @param mixed $reqExtra
     */
    public function setReqExtra($reqExtra): void
    {
        $this->reqExtra = $reqExtra;
    }

    /**
     * @return mixed
     */
    public function getOperator()
    {
        return $this->operator;
    }

    /**
     * @param mixed $operator
     */
    public function setOperator($operator): void
    {
        $this->operator = $operator;
    }


}

?>

File: src/main/java/com/ctrip/ccard/creditcard/vcc/bean/QueryCloseResultRequest.java
package com.ctrip.ccard.creditcard.vcc.bean;


public class QueryCloseResultRequest {
    /**
     * the request number of the original close request number
     */
    private String requestId;
    /**
     * the request time of the original close request time
     */
    private String requestTime;
    /**
     * the merchant name of the original close merchant name
     */
    private String merchantName;
    /**
     * request extra info
     * json type
     * eg: {"BinFlagDcre":"12345"}
     */
    private String reqExtra;
    /**
     * operator
     */
    private String operator;

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public String getRequestId() {
        return requestId;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

    public String getRequestTime() {
        return requestTime;
    }

    public void setRequestTime(String requestTime) {
        this.requestTime = requestTime;
    }

    public String getMerchantName() {
        return merchantName;
    }

    public void setMerchantName(String merchantName) {
        this.merchantName = merchantName;
    }

    public String getReqExtra() {
        return reqExtra;
    }

    public void setReqExtra(String reqExtra) {
        this.reqExtra = reqExtra;
    }
}
